docs/06-concepts/02-models.md

```diff
@@ -226,9 +226,31 @@ Supports any string value, e.g., `"This is a string"` or `'This is a string'`.
 Example:
 
 ```yaml
-stringDefault: String, default='This is a string', defaultModel="This is a string", defaultDatabase="This is a string"
+stringDefault: String, default='This is a string', defaultModel="This is a string", defaultPersist="This is a string"
 ```
 
+### UuidValue
+
+Supports two types of default values:
+
+1. **UUID String Values**:
+   - A valid UUID string can be specified, e.g., `'550e8400-e29b-41d4-a716-446655440000'`.
+
+   Example:
+
+   ```yaml
+   uuidDefault: UuidValue, default='550e8400-e29b-41d4-a716-446655440000', defaultModel='550e8400-e29b-41d4-a716-446655440000', defaultPersist='550e8400-e29b-41d4-a716-446655440000'
+   ```
+
+2. **Random UUID**:
+   - You can use `"random"` to specify a random UUID. On the Dart side, this will be internally handled by `Uuid().v4obj()` to generate a UUID at runtime. On the database side, the `gen_random_uuid()` function is used to generate the UUID.
+
+   Example:
+
+   ```yaml
+   uuidDefaultRandom: UuidValue, default=random, defaultModel=random, defaultPersist=random
+   ```
+
```

Comment:
Question: Could we improve the formatting here? It looks a bit funky in and it is a bit unclear that these are key values. Maybe a table could be used instead? Same for DateTime `now` and string value.

Reply:
I just made the UUID part more clear, let me know if its still not good?!